src/main/java/com/courtcanva/ccfranchise/controllers/StaffController.java

```diff
@@ -0,0 +1,34 @@
+package com.courtcanva.ccfranchise.controllers;
+
+import com.courtcanva.ccfranchise.exceptions.ExpiredVerificationTokenException;
+import com.courtcanva.ccfranchise.exceptions.MailingClientException;
+import com.courtcanva.ccfranchise.services.StaffService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@RequestMapping("/staff")
+@RequiredArgsConstructor
+public class StaffController {
+
+    private final StaffService staffService;
+
+    @PostMapping("/verification")
+    @ResponseStatus(HttpStatus.CREATED)
+    public void sendVerificationEmail(@RequestParam Long id) throws MailingClientException {
+        staffService.sendVerificationEmail(id);
+    }
+
+    @PostMapping("/verify")
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    public void verifyEmail(@RequestParam String token, @RequestParam(name = "i") String email) throws ExpiredVerificationTokenException {
```

Comment:
is this `name = "i"` a deliberate choice?

Reply:
Yup. It was short for `identity`, though..... I didn't put too many thoughts into the name of this request param here. If there is a suggestion there, happy to rename this. Otherwise, I might pick `email` if that suits?

Reply:
This is updated to use `email` for the second `@RequestParam` instead